include/apps/_experiments/weather.h

```diff
@@ -0,0 +1,123 @@
+#pragma once
+#include <osw_hal.h>
+#include <vector>
+#include "osw_app.h"
+
+using namespace std;
+
+class PrintWeatherIcon {
```

Comment:
Could you make this an inner class of `OswAppWeather`? Or an additional header?

Reply:
i think that using an additional header is a god idea, where i should putt all the files? I schould use include/apps/_experiments and src/apps/_experiments for .h and .cpp ?

Reply:
Bingo - this is a good idea.